src/theme.js

```diff
@@ -0,0 +1,28 @@
+const colors = {
+  black: "#262626",
+  white: "#fff",
+  dark_red: "#B3001B",
+  light_blue: "#7ea3cc",
+  mid_blue: "#255c99",
+  mid_green: "#3D583A",
+};
+const backGroundColors = {
+  white: "#fff",
+  black: "#262626",
+  grey: "#999999",
+};
+const fonts = { body: "Roboto, sans-serif" };
+
+const fontSize = [
+  0, 0.25, 0.5, 075, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5, 2.75, 3,
```

Comment:
Tem um errinho aqui, acho que deveria ser `0.75` no lugar de `075`:
```suggestion
  0, 0.25, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5, 2.75, 3,
```
![image](https://user-images.githubusercontent.com/54550926/160854147-62797759-e14a-4171-a1a8-523d14307024.png)